pkg/ceph/client/pool.go

```diff
@@ -127,15 +132,22 @@ func CreatePool(context *clusterd.Context, clusterName string, newPool CephStora
 		}
 	}
 
+	// ensure that the newly created pool gets an application tag
+	args = []string{"osd", "pool", "application", "enable", newPool.Name, appName}
+	buf, err = ExecuteCephCommand(context, clusterName, args)
+	if err != nil {
+		return "", fmt.Errorf("failed to enable application %s on pool %s. %+v", appName, newPool.Name, err)
+	}
+
 	logger.Infof("creating pool %s succeeded, buf: %s", newPool.Name, string(buf))
-	return string(buf), nil
+	return fmt.Sprintf("pool '%s' created", newPool.Name), nil
```

Comment:
actually i moved the message creation up to the API layer (instead of here in the ceph client layer).  i think that's more appropriate and and in accordance with what `block create` does now that i looked again.